nvflare/private/fed/server/server_commands.py

```diff
@@ -293,6 +294,33 @@ def process(self, data: Shareable, fl_ctx: FLContext):
         return errors
 
 
+class ResetErrorsCommand(CommandProcessor):
+    """To implement the show_errors command."""
+
+    def get_command_name(self) -> str:
+        """To get the command name.
+
+        Returns: ServerCommandNames.GET_ERRORS
+
+        """
+        return ServerCommandNames.RESET_ERRORS
+
+    def process(self, data: Shareable, fl_ctx: FLContext):
+        """Called to process the abort command.
+
+        Args:
+            data: process data
+            fl_ctx: FLContext
+
+        Returns: Engine run_info
+
+        """
```

Comment:
nothing is returned here

Reply:
RESET_ERRORS does not return value. Removed.